src/bb/fs/fsclose.c

```diff
@@ -0,0 +1,23 @@
+#include "PR/os_internal.h"
+#include "PR/bbfs.h"
+
+s32 osBbFClose(s32 fd) {
+    BbFat16* fat;
+    s32 rv;
+
+    if ((u32)fd >= BB_INODE16_NUM) {
```

Comment:
```suggestion
    if (fd < 0 || fd >= BB_INODE16_NUM) {
```
This has matched elsewhere and is a bit more natural